docs/source/reference/envvars.rst

```diff
@@ -278,3 +278,16 @@ Threading Control
 
    *Default value:* The number of CPU cores on the system as determined at run
    time, this can be accessed via ``numba.config.NUMBA_DEFAULT_NUM_THREADS``.
+
+.. envvar:: NUMBA_THREADING_LAYER
+
+   This environment variable controls the library used for concurrent execution
+   for the CPU parallel targets (``@vectorize(target='parallel')``, 
+   ``@guvectorize(target='parallel')``  and ``@njit(parallel=True)``). The 
+   variable type is string and by default is ``workqueue``, the Numba internal
```

Comment:
You discuss `default` below, so probably it's out of sync here

Reply:
Thanks, not updated this since the new criteria based backend selection was added. Will change.